### DISCUSSION TITLE
Is Fan Control limited to a total of 6 fans headers?

### DISCUSSION THREAD

**Discussion by No-good-deed-goes-unpunished** [Rem0o/FanControl.Releases#2165] [Q&A]
My motherboard is an MSI X299 XPower Gaming AC. It has 10 fan headers: Pump_fan, CPU_Fan, 4 x Sys fans and 4 x EXS Fans.
Fan Cotrnol will not detect fans connected to any of the EXS_FANx headers.

I've removed my Corsair Commander Pro as the fans on that were sometimes being detected, sometimes not, so I suspect the device may be faulty.

However, my H115i RGB Pro XT (CW-9060044-WW), cooler is never detected by assisted setup, even though the corsairlink plugin is installed. Since it's not detected I can't use the Coolant temp as a source.

**Comment by Rem0o** [comment]
FanControl has no "N" limitation. You could have a 100 control.  In your case, your EXS fan headers are most likely on a second control chip on your mother board which is not detected by the backend library (LHM).

**Comment by No-good-deed-goes-unpunished** [comment]
Ok, I'll move the fans on tyhe EXS headers to SYS headers using splitters and run two fans per header.
